File: spoj/3410.c
#include <stdio.h>

int main() {
    int side;
    while (1) {
        scanf("%d", &side);
        if (side < 1)
            break;
        int result = 0;
        int counter = 1;
        int n = side;
        for (;n > 0;n--) {
            result += (n*counter);
            counter += 2;
        }
        printf("%d\n", result);
    }
}

File: coding/spoj/3410.go
package main

import "fmt"

func main() {
	var side int
	for {
		fmt.Scanf("%d", &side)
		if side == 0 {
			break
		}
		if side == 1 {
			fmt.Println(1)
		}
		result := 0
		counter := 1
		n := side
		// sides = 1*nC1 + 3*n-1C1 + 5*n-2C1...
		for ; n > 0; n-- {
			result += n * counter
			counter += 2
		}
		fmt.Println(result)
	}

}
